e3tools/local-development-portal
# Update village detail page (first version)

Update admin level detail page  (first version)

## How to test?
- [ ] Go to village page 
- Check new distribution of information
- Check priority table new design

- [ ] Go to village page 
- Check new distribution of information
- Check priority table new design

- [ ] Go to canton page 
- Check new distribution of information
- Check priority table new design

- [ ] Go to region page 
- Check new distribution of information
- Check priority table new design

- [ ] Go to prefecture page 
- Check new distribution of information
- Check priority table new design

- [ ] Go to search admin level page 
- Search for village with dropdowns and check that navigation works
